.github/workflows/wasm_demos.yaml

```diff
@@ -10,6 +10,7 @@ on:
 jobs:
   wasm_demo:
     env:
+      RUSTFLAGS: "--cfg=web_sys_unstable_apis"
```

Comment:
It would be nice to keep one CI job that do not have this.  So i'd put it only for nightly jobs (that build the artifacts). but not for the regular CI (so we can test that we don't break user code that don't set this flag)
But you can leave it as is for now i can adjust the CI later.

Reply:
I think you know it better . prefer you fix it this one =)